Integrations/integration-Zendesk.yml

```diff
@@ -140,6 +140,12 @@ script:
         }
     };
 
+    var getOrganizationDetails = function(id) {
+        var res = sendRequest('GET', 'organizations/' + id + '.json');
+        var org = res.organization;
```

Comment:
too many assignments.
Just do:
`return res.organization `
Or even 
`sendRequest('GET', 'organizations/' + id + '.json') .organization`

Reply:
yes. you are right. had there more logic which I removed so it stayed... will do the return res.organization.